eatsmart/locations/wake/management/commands/import_wake.py

```diff
@@ -114,6 +117,21 @@ def save_inspections(self, inspections):
             inspection_obj.__dict__.update(**attributes)
             inspection_obj.save()
 
+    def get_risk_factor_value(self, risk_factor):
+        risk_factor_dict = {
+            'Food from Unsafe Sources': 5,
+            'Improper Holding': 1,
+            'Food from Unsafe Source': 5,
+            'Poor Personal Hygiene': 4,
+            None: 6,
+            'Contaminated Equipment': 3,
+            'Inadequate Cook': 2
+        }
+        try:
```

Comment:
it looks cleaner if you return this instead:

return risk_factor_dict.get(key, 0)  # this never raises an exception and if the key is not found it returns 0.